README.md

```diff
@@ -1,786 +1,104 @@
+This is what a common `CITATION.cff` file for research software may look like:
 
 ```yaml
+cff-version: 1.2.0
+message: If you use this software, please cite it using these metadata.
+title: My Research Software
+abstract: This is my awesome research software. It does many things.
+authors:
+  - family-names: Druskat
+    given-names: Stephan
+    orcid: "https://orcid.org/0000-0003-4925-7248"
+version: 0.11.2
+date-released: "2021-07-18"
 identifiers:
-  - type: "other"
-    value: "my-custom-identifier-1234"
+  - description: This is the collection of archived snapshots of all versions of My Research Software
+    type: doi
+    value: "10.5281/zenodo.123456"
+  - description: This is the archived snapshot of version 0.11.2 of My Research Software
+    type: doi
+    value: "10.5281/zenodo.123457"
+license: Apache-2.0
+repository-code: "https://github.com/citation-file-format/my-research-software"
 ```
 
-# Specified value strings
+## Format specifications :books:
 
-The keys `status`, `license`, `languages`, and `identifier:type` can only take values
-from a fixed set of strings. These are specified below.
+You can find the complete format specifications in the [Guide to the Citation File Format schema](schema-guide.md).
 
-## Status strings
+## Why should I add a `CITATION.cff` file to my repository? :bulb:
 
-Works can have a different status of publication, e.g., journal papers. CFF
-specifies the following value strings for the key `status`.
+When you do this, great things may happen:
 
-  | Status (String)  |                                     Description                                      |
-  |------------------|--------------------------------------------------------------------------------------|
-  | `in-preparation` | A work in preparation, e.g., a manuscript (covers drafts)                            |
-  | `abstract`       | The abstract of a work                                                               |
-  | `submitted`      | A work that has been submitted for publication                                       |
-  | `in-press`       | A work that has been accepted for publication but has not yet been published         |
-  | `advance-online` | A work that has been published online in advance of publication in the target medium |
-  | `preprint`       | A work that has been published as a preprint before peer review                      |
+1. People that find your software can easily cite it using the correct metadata from `CITATION.cff`!
+2. If your repository is hosted on GitHub, [they will link to it](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-citation-files) in a special "Cite this repository" widget on the repository landing page. Also, the citation information in `CITATION.cff` will be used when you publish your software on Zenodo via the [GitHub-Zenodo bridge](https://guides.github.com/activities/citable-code/).
```

Comment:
- "they will link to it" not a convincing argument, maybe something like _they will show a widget in the sidebar that makes it easy for visitors to cite your software correctly_
- github-zenodo bridge part could be a separate point